docs/astrocut/index.rst

```diff
@@ -142,45 +142,49 @@ treated as the R, G, and B channels respectively.
 TESS Full-Frame Image Cutouts
 =============================
 
-There are two parts of the package involved in this task, the `~astrocut.CubeFactory`
-class allows you to create a large image cube from a list of FFI files.
+There are two parts of the package involved in creating cutouts from TESS full-frame images (FFIs).
+First, the `~astrocut.CubeFactory` (if working with SPOC products, or `~astrocut.TicaCubeFactory` if working
+with TICA FFIs) class allows you to create a large image cube from a list of FFI files.
 This is what allows the cutout operation to be performed efficiently.
-The `~astrocut.CutoutFactory` class performs the actual cutout and builds
-a target pixel file (TPF) that is compatible with TESS pipeline TPFs.
-
-The basic work-flow is to first create an image cube from individual FFI files
-(this is one-time work), and then make individual cutout TPFs from this
-large cube file. If you are doing a small number of cutouts, it may make
-sense for you to use our tesscut web service:
-`mast.stsci.edu/tesscut <https://mast.stsci.edu/tesscut/>`_
+Next, the `~astrocut.CutoutFactory` class performs the actual cutout and builds
+a target pixel file (TPF) that is similar to the TESS Mission-produced TPFs.
+
+The basic procedure is to first create an image cube from individual FFI files
+(this only needs to be completed once per set of FFIs), and to then make individual cutout TPFs from this
+large cube file for targets of interest. If you are creating a small number of cutouts, the TESSCut web service
+may suit your needs: `mast.stsci.edu/tesscut <https://mast.stsci.edu/tesscut/>`_
  
 Making image cubes
 ------------------
 
-Making an image cube is a simple operation, but comes with an important
-time/memory trade-off.
-
 .. important::
-   **Time/Memory Trade-off**
+   **Time-Memory Trade-off**
+
+   Making an image cube is a simple operation, but comes with an important
+   time-memory trade-off.
 
    The ``max_memory`` argument determines the maximum memory in GB that will be used
-   for the image data cube while it is being built. This is *only* for the data cube,
-   and so is somewhat smaller than the amount of memory needed for the program to run.
-   Never set it to your system's total memory.
+   for the image data cube while it is being built. This is the amount of memory required 
+   *only* for the data cube, so is somewhat smaller than the total amount of memory needed
+   for the program to run. You should never set it to your system's total memory.
 
    Because of this, it is possible to build cube files with much less memory than will
-   hold the final product. However there is a large time trade-off, as the software must
+   hold the final product. However, there is a large time trade-off, as the software must
```

Comment:
The wording of this first sentence seems odd to me. Thoughts on how to re-word this?

Reply:
How about something like this: "Because of this, cube files do not need to allocate their total size in memory all at once. Instead, a smaller memory allocation can be used while the cube file is constructed, however, this will significantly increase the execution time as bytes are swapped into and out of the memory allocation being used."

Reply:
That sounds better to me! Adding this change.